테스트 DB 설정, 음식점 정보 수정 테스트

SOULIKK opened the pull request:
테스트 환경에서 사용할 h2 db 설정과 음식점 정보 수정 기능 단위 테스트를 추가했습니다

SOULIKK committed:
FEAT: 테스트 DB 설정, 음식점 정보 수정 테스트

SteadyFlow736 reviewed (changes requested):
라이언님이 말씀하셨던 것 처럼,
StoreService에 대한 테스트라면, StoreService 객체에 대한 테스트로 변경되어야 하고 repository도 mocking으로 처리되어야 할 것 같습니다.

SOULIKK committed:
REFACTOR: 음식점(Store) 정보 수정 기능 수정
Address 클래스에 StoreUpdateDto 참조되는 부분 제거했습니다.
수정된 음식점의 id값을 리턴하도록 수정했습니다.
StoreUpdateDto의 클래스명을 UpdateStoreRequest로 수정했습니다.
UpdateStoreRequest에서 Seller 객체가 아닌 sellerId를 받도록 수정했습니다.

SOULIKK committed:
REFACTOR: 음식점(Store) 정보 수정 기능 테스트
@SpringBootTest를 포함한 Repositoty 테스트에서 Mockito를 사용한 유닛테스트로 수정하였습니다.
